website/versioned_docs/version-0.67/integration-with-android-fragment.md

```diff
@@ -51,8 +99,29 @@ public class MyReactApplication extends Application implements ReactApplication
 }
 ```
 
+</TabItem>
+</Tabs>
+
 If you are using Android Studio, use Alt + Enter to add all missing imports in your class. Alternatively these are the required imports to include manually:
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="kotlin">
+
+```kotlin
+import android.app.Application
+
+import com.facebook.react.PackageList
+import com.facebook.react.ReactApplication
+import com.facebook.react.ReactNativeHost
+import com.facebook.react.ReactPackage
+import com.facebook.soloader.SoLoader
+
+import java.util.List
```

Comment:
This should be removed. When on Kotlin you should use Kotlin std-lib's type wrappers.